visualization/index.html

```diff
@@ -73,25 +85,23 @@ <h3>ASL-LEX 2.0</h3>
             </div>
         </div>
 
-        <ul class="list-unstyled CTAs" id="description">
-            This visualization allows us to view the Lexicon as a network graph. Each circular represents a sign. Click on a sign to view properties.
-                You can search for a sign using the search bar, or filter signs of interest by using the filter option below. You can also select a subset of signs by dragging your mouse cursor over them.
-
-        </ul>
         <ul class="list-unstyled CTAs">
 
-            <li>Search For A Word</li>
+            <li style="padding-top: 20px">Search For A Word<i class="fas fa-info-circle fa-info-circle-padding-right" data-toggle="tooltip" data-placement="top" title="Type an English word to find the corresponding sign on the graph"></i></li>
```

Comment:
Same goes for the style here

Reply:
For line 90 and 95 below, can we declare them in a js file like is done for filters_data.js and use handlebars to render it? That ways, if anyone had to change the tooltip text they wouldn't be lost in the html to find it